### ISSUE TITLE
default driver GPKG for fiona 2.0?

### ISSUE THREAD

**Issue by snowman2** [Toblerity/fiona-rfc#4]
Thoughts on making GPKG the default driver for writing in fiona 2.0?

http://switchfromshapefile.org/#geopackage

**Comment by hobu** [comment]
My thoughts are generally negative on this. Can you concisely make the case for it for Fiona users?

**Comment by snowman2** [comment]
Currently the default is the ESRI Shapefile.There is a long list of downsides of the shapefile mentioned here: http://switchfromshapefile.org/

On the same page, about the geopackage (http://switchfromshapefile.org/#geopackage) they say:

> OGC GeoPackage is one of the most promising formats, designed for today's modern applications. GeoPackage is published as standard by the Open Geospatial Consortium. 

One plus that I didn't find there was that I have found that the geopackage works when reading it using `/vsis3/` remotely with fiona. I have attempted doing the same with the shapefile by zipping it up, but without success. I could read in the file locally, but not on the cloud.
